usr/share/cachyos-hyprland-settings/variables.conf

```diff
@@ -0,0 +1,133 @@
+# ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+# ┃                    Variables Configuration                  ┃
+# ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+
+source = /usr/share/cachyos-hyprland-settings/colors.conf
+
+# Variables wiki https://wiki.hyprland.org/0.45.0/Configuring/Variables/ #
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#general
+general {
+
+    gaps_in = 3
+    gaps_out = 5
+    border_size = 3
+    col.active_border = $cachylgreen
+    col.inactive_border = $cachymblue
+    layout = dwindle # master|dwindle
+
+    # https://wiki.hyprland.org/0.45.0/Configuring/Variables/#snap
+    snap {
+    	enabled = true
+    	window_gap = 10
+    	monitor_gap = 10
+    	border_overlap = false
+    }
+
+}
+
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#gestures
+gestures {
+     workspace_swipe = true
+     workspace_swipe_fingers = 4
+     workspace_swipe_distance = 250
+     workspace_swipe_invert = true
+     workspace_swipe_min_speed_to_force = 15
+     workspace_swipe_cancel_ratio = 0.5
+     workspace_swipe_create_new = false
+}
+
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#group
+group {
+	auto_group = true
+	insert_after_current = true
+	focus_removed_window = true
+	col.border_active = $cachydgreen
+	col.border_inactive = $cachylgreen
+	col.border_locked_active = $cachymgreen
+	col.border_locked_inactive = $cachydblue
+
+    # https://wiki.hyprland.org/0.45.0/Configuring/Variables/#groupbar
+	groupbar {
+		enabled = true
+		font_family = "JetBrainsMono Nerd Font"
+		font_size = 8
+		text_color = $cachydblue
+		col.active = $cachydgreen
+		col.inactive = $cachylgreen
+		col.locked_active = $cachymgreen
+		col.locked_inactive = $cachydblue
+	}
+}
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#misc
+misc {
+    font_family = "JetBrainsMono Nerd Font"
+    splash_font_family = "JetBrainsMono Nerd Font"
```

Comment:
@Shendisx jetbrains isn't part of the dependencies in the PKGBUILD so this could be breaking. If we're going with this we need to add it to our deps.

Reply:
up to you, but yeah thats true

Reply:
Just copied the latest config and split it up (think they got added with the PR from yesterday)